code/modules/urist/modules/jungle/planet_turfs.dm

```diff
@@ -1,32 +1,32 @@
 /turf/simulated/floor/planet
-	var/animal_spawn_chance = 0
-	var/plants_spawn_chance = 0
-	var/small_trees_chance = 0
-	var/large_trees_chance = 0
-	var/misc_plant_spawn_chance = 0
-	var/trap_spawn_chance = 0
+	var/animal_spawn_chance = 0 //% chance to spawn animals. actual percentage is roughly divided in half bc i was an idiot 9 years ago
+	var/plants_spawn_chance = 0 //%chance to spawn plants from the general plant icon database. this includes fruit bearing plants
+	var/small_trees_chance = 0 //% chance to spawn small trees
+	var/large_trees_chance = 0 //% chance to spawn large trees
+	var/misc_plant_spawn_chance = 0 //misc plant spawn chance - this is a specifically defined type
+	var/trap_spawn_chance = 0 //trap spawns??? idfk. check below for the ancient trap code
 	name = "wet grass"
 	desc = "Thick, long wet grass"
 	icon = 'icons/jungle.dmi'
-	icon_state = "grass1"
-	light_max_bright = 0.4
+	icon_state = "grass1" //what the icon looks like when mapping/ingame if icon_spawn_state is not set
+	light_max_bright = 0.4 //these keep tiles lit. you can adjust this to make things more dim or remove lighting entirely.
 	light_inner_range = 0.1
 	light_outer_range = 1.5
 	light_falloff_curve = 0.5
 	footstep_type = /singleton/footsteps/grass
-	var/icon_spawn_state = "grass1"
-	var/farmed = 0
+	var/icon_spawn_state = "grass1" //what the icon looks like when it spawns. if this is not set it defaults to icon_state
+	var/farmed = 0 //has someone tilled this soil?
```

Comment:
_Very_ minor nit-pick, but if this is a yes/no sort of variable, it should be a boolean.
Although functionally in DM `TRUE` and `1` are the same, as well as `FALSE` and `0`; Just makes for better readability to show intent (ie, this variable is `TRUE` or `FALSE`, not some sort of integer counting how many times it's been farmed)
```suggestion
	var/farmed = FALSE //has someone tilled this soil?
```

Reply:
it's not a yes/no variable because it's from 2014 and i was a dumbass. See line 136-157